virt/virt-4-11-release-notes.adoc

```diff
@@ -27,22 +27,21 @@ include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
 //CNV-16390 Supported guest operating systems
 To view the supported guest operating systems for {VirtProductName}, refer to link:https://access.redhat.com/articles/973163#ocpvirt[Certified Guest Operating Systems in Red Hat OpenStack Platform, Red Hat Virtualization and OpenShift Virtualization].
 
-[id="virt-4-11-inclusive-language"]
-== Making open source more inclusive
-
-Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
+include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
```

Comment:
Totally optional suggestion: I know this is where it was before (clearly), but the placement feels awkward to me. I wonder if we should put it at the end of the RNs?

Reply:
I agree that the current placement is not the best. How about putting it at the top instead? I am afraid it might get kinda lost at the end of the RNs. Just my perspective.

Reply:
You're probably right; the top sounds good to me. :)

Reply:
haha -- that's why I'm working on a [solution](https://github.com/openshift/openshift-docs/pull/48293) to add content to the beginning of the CP docs. IIRC, only a handful of the OCP docs include the inclusive language section. They also put it in the RN because there is no easy way to use a module without including it in an assembly. Normally, that assembly would be the master.adoc file. However, the master.adoc files are generated by a Python script, so that's where content like this needs to be injected. If you come up with a good solution, please let me know!